flow-typed/forms.js

```diff
@@ -354,6 +356,14 @@ declare type CivicaStreetNameElement = {
   placeholderValue?: string,
 } & LookupFormElement
 
+declare type CivicaNameRecordElement = {
+  type: 'civicaNameRecord',
+  readOnly: boolean,
+  defaultValue?: CivicaNameRecord,
+  placeholderValue?: string,
+  useGeoscapeAddressing: boolean,
+} & LookupFormElement
```

Comment:
I dont think can allow lookup element here. This element will actually be a bunch of elements. Which one would trigger the lookup?